### DISCUSSION TITLE
Can’t open GitHub on my MacBook

### DISCUSSION THREAD

**Discussion by ybq22** [community/community#138543] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I did the diagnose of GitHub-debug, and the error information is below:
Error downloading
2507998 bytes downloaded from cloud.githubusercontent.com at 3.58 Mbps
2507998 bytes downloaded from avatars.githubusercontent.com at 59.06 Mbps
2507998 bytes downloaded from github.githubassets.com at 30.13 Mbps
2507998 bytes downloaded from australiaeast.github-debug.com at 16.35 Mbps
2507998 bytes downloaded from brazilsouth.github-debug.com at 5.46 Mbps
2507998 bytes downloaded from centralindia.github-debug.com at 25.34 Mbps
2507998 bytes downloaded from fra.github-debug.com at 4.82 Mbps
2507998 bytes downloaded from iad.github-debug.com at 7.30 Mbps
2507998 bytes downloaded from japaneast.github-debug.com at 19.91 Mbps
2507998 bytes downloaded from koreacentral.github-debug.com at 17.22 Mbps
2507998 bytes downloaded from sea.github-debug.com at 8.38 Mbps
2507998 bytes downloaded from southafricanorth.github-debug.com at 9.39 Mbps
2507998 bytes downloaded from southeastasia.github-debug.com at 57.46 Mbps
2507998 bytes downloaded from github-cloud.s3.amazonaws.com at 2.15 Mbps

                
137.132.26.179

                
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.2.1 Safari/605.1.15

                
29788a8d2749ec40781431f3f32d8bef

                
2E06:9FACC:116F4:23DA53:66E3E624

                
                

                  
traceroute to 137.132.26.179 (137.132.26.179), 30 hops max, 60 byte packets
 1  *
 2  *
 3  *
 4  *
 5  *
 6  *
 7  *
 8  *
 9  *
10  *
11  *
12  *
13  *
14  *
15  *
16  *
17  *
18  *
19  *
20  *
21  *
22  *
23  *
24  *
25  *
26  *
27  *
28  *
29  *
30  *

                  
PING 137.132.26.179 (137.132.26.179) 56(84) bytes of data.

--- 137.132.26.179 ping statistics ---
3 packets transmitted, 0 received, 100% packet loss, time 2046ms

                  
Start: 2024-09-13T00:18:07-0700
HOST: dd39278837d3d00577ad50a5aa19e043                Loss%   Snt   Last   Avg  Best  Wrst StDev

                  
Host 179.26.132.137.in-addr.arpa. not found: 3(NXDOMAIN)

**Comment by Sin-cy** [comment]
You mean on the github desktop app ?

**Comment by meower1** [comment]
Try changing your dns and cleaning the isp cache using this command
`sudo dscacheutil -flushcache; sudo killall -HUP mDNSResponder`
